dashboard/src/deployment/DeploymentDetails.js

```diff
@@ -0,0 +1,72 @@
+import ReactTimeout from 'react-timeout';
+import React, { Component } from 'react';
+import api from '../api/api.js';
+import Loading from '../util/Loading.js';
+import MemberList from './MemberList.js';
+import styled from 'react-emotion';
+import { Loader } from 'semantic-ui-react';
+
+const LoaderBox = styled('span')`
+  float: right;
+  width: 0;
+  padding-right: 1em;
+  margin-right: 1em;
+  margin-top: 1em;
+  max-width: 0;
+  display: inline-block;
+`;
+
+const MemberGroupsView = ({memberGroups, namespace}) => (
+  <div>
+    {memberGroups.map((item) => <MemberList 
+      key={`server-group-${item.group}`}
```

Comment:
Any particular reason you're prefixing the keys? They're local to their parent so this doesn't seem to do much.

Reply:
fixed